sdk/python/README.md

```diff
@@ -1,3 +1,91 @@
 # Native Google Cloud Pulumi Provider (preview)
 
-The native Google Cloud Provider for Pulumi lets you use Google Cloud resources in your cloud programs.
+The native Google Cloud Provider for Pulumi lets you provision Google Cloud resources in your cloud programs.
```

Comment:
Is there a requirement to have an sdk/python/README.md? Wasn't sure why we create it generally.

Reply:
The Python README is the one that shows in pypi - so we *generally* copy it 🤷